src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue24856.cs

```diff
@@ -17,9 +17,20 @@ public Issue24856(TestDevice device)
 		[Category(UITestCategories.ImageButton)]
 		public void ImageButtonAspectFitWorks()
 		{
-			App.WaitForElement("WaitForStubControl");
-			App.Tap("UpdateAspect");
-			VerifyScreenshot();
+			try
+			{
+				App.ToggleSystemAnimations(false);
+
+				App.WaitForElement("WaitForStubControl");
+				App.Tap("UpdateAspect");
+
+				Thread.Sleep(1000);
```

Comment:
Can we maybe just take the issue changes for this PR? 

VertifyScreenShot already has some code that retries after a delay

Maybe we just add some code into it for Android that tries against after a slightly longer delay. That way the solution is a bit general?